Dockerfile

```diff
@@ -17,4 +17,4 @@ RUN python -m pytest
 
 # Run the web service on container startup.
 ENV FLASK_ENV="production"
-CMD exec gunicorn --bind :$PORT --workers 1 --threads 8 --timeout 0 main:app
\ No newline at end of file
+CMD exec gunicorn --bind :8080 --workers 1 --threads 8 --timeout 0 main:app
```

Comment:
Was this intentional (or should it be kept as $PORT)?

Reply:
As the PORT not need to change so I made it constant in the container and removed the PORT var in https://github.com/datacommonsorg/website/pull/38/commits/fca37637a0ee322592b471a417ff948a85bfcb19#diff-04c6e90faac2675aa89e2176d2eec7d8L60.